mx.py

```diff
@@ -2222,6 +2222,9 @@ def resolveDeps(self):
         if licenseId:
             self.theLicense = get_license(licenseId, context=self)
 
+    def substVars(self, text):
+        return text.format(**vars(self))
```

Comment:
Yes, I understand. It's just that you need to understand you are referring to instance variables which a) may not reflect any attributes in suite.py or b) may have values different from those in suite.py (e.g. a "path" attribute may have a relative path value and a `path` instance variable may be the absolute path derived from that relative path).

This should be added as documentation to `substVars`.